### PR TITLE
feat(devtools): click slot name collapse/expand styles within

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
Clicking on slot name collapse/expand styles within:
Screenshot 2022-05-03 at 16 12 22

### CODE REVIEW

File: packages/devtools/src/SlotCSSRules.tsx [hunk 1]
```diff
@@ -25,15 +26,22 @@ export const SlotCSSRules: React.FC<{ slot: string; atomicRules: AtomicRules[] }
   const rules = React.useMemo(() => getMonolithicCSSRules(atomicRules), [atomicRules]);
   const classes = useStyles();
 
+  const [expanded, setExpanded] = React.useState(true);
+  const handleSlotNameClick = () => setExpanded(v => !v);
```
Comment (ling1726) [inline packages/devtools/src/SlotCSSRules.tsx:30]:
```suggestion
  const toggleExpanded= () => setExpanded(v => !v);
```

**Review by ling1726 — APPROVED** [review]

File: packages/devtools/src/SlotCSSRules.tsx [hunk 2]
```diff
@@ -25,15 +26,22 @@ export const SlotCSSRules: React.FC<{ slot: string; atomicRules: AtomicRules[] }
...
 
   return (
     <>
-      <pre className={classes.slotName}>{slot}</pre>
-      <div className={classes.rules} onClick={undoHighlight}>
-        <MonolithicRulesView rules={rules} />
-      </div>
+      <pre className={classes.slotName} onClick={toggleExpanded}>
+        {slot}
+      </pre>
```
Comment (layershifter) [inline packages/devtools/src/SlotCSSRules.tsx:39]: I think that are missing there an indicator to indicate if it's opened:

image

Comment (YuanboXue-Amber) [reply packages/devtools/src/SlotCSSRules.tsx:39]: good idea, let me create a separate PR for this

Comment (YuanboXue-Amber) [reply packages/devtools/src/SlotCSSRules.tsx:39]: PR #116

**Pull request merged**